cmd/terramate/cli/cli.go

```diff
@@ -1743,19 +1758,41 @@ func (c *cli) printStacksList(allStacks []stack.Entry, why bool, runOrder bool)
 	}
 }
 
-func parseStatusFilter(strStatus string) cloudstack.FilterStatus {
-	status := cloudstack.NoFilter
-	if strStatus != "" {
-		status = cloudstack.NewStatusFilter(strStatus)
-		if status.Is(cloudstack.Unrecognized) {
-			fatalf("unrecognized stack filter: %s", strStatus)
-		}
+func parseStatusFilter(filterStr string) cloudstack.FilterStatus {
+	if filterStr == "" {
+		return cloudstack.NoFilter
+	}
+	filter, err := cloudstack.NewStatusFilter(filterStr)
+	if err != nil {
+		fatalWithDetails(err, "unrecognized stack filter")
```

Comment:
I see  this is where my earlier  comment regarding the different error wording is coming from. I think they should all be aligned.